esquery.js

```diff
@@ -55,8 +55,8 @@ function inPath(node, ancestor, path) {
     const field = ancestor[path[0]];
     const remainingPath = path.slice(1);
     if (Array.isArray(field)) {
-        for (let i = 0, l = field.length; i < l; ++i) {
-            if (inPath(node, field[i], remainingPath)) { return true; }
+        for (const fld of field) {
```

Comment:
This is not a great variable name. `component`?

Reply:
Sure, SGTM. Added.